### ISSUE TITLE
More Metadata?

### ISSUE THREAD

**Issue by hg2s** [agersant/polaris#134]
Would it be possible to view other track metadata? For example, release date, composer, record label, barcode, catalog number, genre. 

-Thank you

**Comment by agersant** [comment]
A couple questions that could help getting this feature right (one day):
- What format do you use to encode said metadata? (I assume ID3v2)
- Do you usually look this up for songs you are currently listening to, or while browsing the collection looking for something to play?

**Comment by hg2s** [comment]
> * What format do you use to encode said metadata? (I assume ID3v2)

Yes, exactly. I'm thinking of ID3v2. I'm using [Beets](https://beets.io/) quite a bit where ID3v2.4 is the current default. 

> * Do you usually look this up for songs you are currently listening to, or while browsing the collection looking for something to play?

A typical use case is browsing the collection looking for a specific recording to play or while building a playlist. 

Since Polaris is so good at handling large collections I'm thinking it would be especially useful in musical archives, radio stations and libraries. To that end I imagine an Advanced Search option that offers various metadata as search fields.

**Comment by agersant** [comment]
✅ Implemented in polaris 0.15.0
